_rules/role-attribute-valid-value-674b10.md

```diff
@@ -37,9 +37,14 @@ This rule applies to any `role` attribute for which all the following are true:
 - the attribute has a value that is neither empty ("") nor only [ASCII whitespace][]; and
 - the attribute is specified on an [HTML or SVG element][] which is not [programmatically hidden][].
 
-## Expectation
+## Expectation (mapping to 4.1.2)
+
+Each test target either has at least one token which is a valid value corresponding to a non-abstract role from [WAI-ARIA Specifications][]; or the test target is not a [user interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components)
```

Comment:
I think both the UIC and the structure definitions are ambiguous. UIC because it is not clear who is meant by "users", and "structure" for not specifying what "organize" means.

Reply:
We can replace UIC by our definition of [Instrument](https://act-rules.github.io/glossary/#instrument-to-achieve-an-objective) but that would require to define the objective, so I'm not sure it would actually work 😕